src/Stripe.net/Services/Disputes/DisputeService.cs

```diff
@@ -22,14 +22,14 @@ public DisputeService(IStripeClient client)
 
         public override string BasePath => "/v1/disputes";
 
-        public virtual Dispute Close(string id, RequestOptions requestOptions = null)
+        public virtual Dispute Close(string id, DisputeCloseOptions options = null, RequestOptions requestOptions = null)
```

Comment:
@ob-stripe I never know if this change is a breaking change

Reply:
I think this is a breaking change since we're adding a new argument. If someone was calling this before with request options as the second arg, it'll fail because we're changing the type for the second arg.